### ISSUE TITLE
Craft 5 Support

### ISSUE THREAD

**Issue by gateszies** [dispositiontools/craft-alt-text-generator#3]
Our team is loving the functionality of this app. Would you happen to have plans to upgrade for Craft 5 compatibility? It's currently marked as `Not Ready` in the Craft 5 Upgrade panel.

**Comment by dispositiontools** [comment]
Thanks for comment! 

I am working on the Craft5 support right now - should be ready for Monday. 

I foolishly tried to includes some needed imrpovements in to the same update which has meant it's taken a little longer but almost there.

**Comment by gateszies** [comment]
Amazing! Thanks for the update.

After playing around a bit, a few things we noticed that would make things easier for our authors:
- Automatically fetch generated alt text without having to select `Refresh details` for each image
- Edit suggested alt text inline before Syncing to asset
- Prompt or button to generate alt text in the asset vs. selecting from table

Again, we're really enjoying the integration of the tool with this plugin. Appreciate the work you've done to build it out!

**Comment by dispositiontools** [comment]
Hi @gateszies I've pushed two updates today. 

I've been working on a couple of improvements at the same time as doing the update and that includes editing the alt text before syncing. 

It's also added some error reporting for when things go wrong, and for allowing for resubmitting the image for alttext.ai to re-create the alt text. 

I've also added the ability to change which language model is being used, and which language it's returned in.

The plugin does normally fetch the generated al text immediately unless you have the "async" option turned on in the settings - in which case you'll prob need the make sure the queue is being automatically run. I had had the async option as on as default which I've now updated. 

I will be adding the prompt button the assets entry page soon!

Thank you so much for your comments - I'm looking forward to making it better
